### ISSUE TITLE
Bad translation or bad display of the message

### ISSUE THREAD

**Issue by LoakAudio** [prusa3d/Prusa-Firmware#4424]
**Printer type** - MK3S+ no MMU
**Printer firmware version** -  3.13.1- French language selected

**Describe the bug**
 During a filament change, when it's time to introduce the new filament the message showed is not clear, I think last letters of the message are mixed at the start of it ( see photo)

![Message Prusa](https://github.com/prusa3d/Prusa-Firmware/assets/39970118/da9567fe-8b92-44b3-a00e-70978f72d81a)

In addition, when introduced, the filament loads automatically, no need to push a button as it seems to be said on the screen
  
**To Reproduce**
Try a Print with firmware 3.13.1 in french, no MMU, with a filament change 

  
**Expected behavior**
 A showed message describing which action is needed
  
Don't hesitate if you need more information
Thanks !

**Comment by 3d-gussner** [comment]
@LoakAudio Thanks for reporting the issue. Yes in French the line is too long `et appuyez sur le bouton` and isn't shown correctly on LCD screen
```
 |01234567890123456789|
0|                    |
1|Inserez le filament |
2|et appuyez sur le bo|uton
```
And the `uton` overwrite part of the `Inserez`

**Comment by 3d-gussner** [comment]
@LoakAudio Merci beaucoup again I created a fix in #4543 and it looks like this
![Fix_4424](https://github.com/prusa3d/Prusa-Firmware/assets/25530011/74d0a7ef-0a2b-49f6-b63a-df61f94bee0d)

I am closing this issue. If you disagree please reopen it again.
